doc/nrf/libraries/caf/sensor_data_aggregator.rst

```diff
@@ -0,0 +1,79 @@
+.. _caf_sensor_data_aggregator:
+
+CAF: Sensor data aggregator module
+##################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The |sensor_data_aggregator| of the :ref:`lib_caf` (CAF) is a simple module responsible for aggregating sensor data in form of ``sensor_event`` and passing them further in packages.
+It can be used in multi-core chips to reduce power consumption.
+In this case, one core gathers data from sensors and when there is sufficient data to analyze, the first core wakes up the other core and sends the aggregated data to that core.
+
+Configuration
+*************
+
+The |sensor_data_aggregator| module can be enabled by selecting the :kconfig:option:`CONFIG_CAF_SENSOR_DATA_AGGREGATOR` option.
+
+To use the module, you must complete the following steps:
+
+1. Enable the :kconfig:option:`CONFIG_CAF_SENSOR_DATA_AGGREGATOR` option.
+#. Enable the :kconfig:option:`CONFIG_CAF_SENSOR_DATA_AGGREGATOR_EVENTS` option, If you want to receive aggregated data on another core.
+#. Enable aggregator in devicetree file that describes the aggregator parameters.
+   Each aggregator should be placed as a separate node.
+   For example, the file content could look like follows:
+
+   .. code-block:: devicetree
+
+      agg0: agg0 {
+              compatible = "caf,aggregator";
+              sensor_descr = "accel_sim_xyz";
+              buf_data_length = <120>;
+              sensor_data_size = <12>;
+              status = "okay";
+      };
+
+      agg1: agg1 {
+             compatible = "caf,aggregator";
+             sensor_descr = "void_test_sensor";
+             buf_data_length = <80>;
+             sensor_data_size = <8>;
+             status = "okay";
+      };
+
+Two aggregators are defined here and each one is responsible to handle data of the different sensors. The aggregator is defined as a separate node in the devicetree and consists of the following parameters:
+
+* ``compatible`` - This is DTS binding and should be set to ``caf,aggregator``.
```

Comment:
I was about to ask if the comma in the file name is intended, but I guess it is...